copyright

```diff
@@ -282,13 +282,23 @@ Files:
  images/ship/hai?centipede*
  images/ship/hai?geocoris*
  images/ship/hai?grasshopper*
+ images/ship/gull*
+ images/ship/pelican*
  images/thumbnail/hai?centipede*
  images/thumbnail/hai?geocoris*
  images/thumbnail/hai?grasshopper*
+ images/thumbnail/gull*
+ images/thumbnail/pelican*
 Copyright: Evan Fluharty (Evanfluharty@gmail.com)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (from under the same license).
 
+Files:
```

Comment:
With the removal of the Salvage Scanner/Scanner Software, this entry can now be removed.

Reply:
removed.